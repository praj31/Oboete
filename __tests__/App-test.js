// /**
//  * @format
//  */

// import 'react-native';
// import React from 'react';
// import App from '../App';

// // Note: test renderer must be required after react-native.
// import renderer from 'react-test-renderer';

// it('renders correctly', () => {
//   renderer.create(<App />);
// });

const sum = require('../utils/sum');

test('adds 1 + 2 to equal 3', () => {
  expect(sum(1, 2)).toBe(3);
});