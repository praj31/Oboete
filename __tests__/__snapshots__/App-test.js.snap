// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test: Form should be empty after clicking cancelling 1`] = `
<View>
  <TextInput
    maxLength={40}
    onChangeText={[Function]}
    testID="reminder-title"
    value=""
  />
  <TextInput
    onChangeText={[Function]}
    testID="reminder-date"
    value=""
  />
  <TextInput
    onChangeText={[Function]}
    testID="reminder-time"
    value=""
  />
  <TextInput
    onChangeText={[Function]}
    testID="reminder-interval"
    value=""
  />
  <TextInput
    onChangeText={[Function]}
    testID="reminder-repeat"
    value=""
  />
  <View
    accessibilityRole="button"
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        [
          {},
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Cancel
      </Text>
    </View>
  </View>
  <Text
    testID="printed-test"
  >
    Pass
  </Text>
</View>
`;
